src/main.js

```diff
@@ -93,12 +90,17 @@ if (
 
 app.component("inline-svg", InlineSvg);
 app.use(i18n);
-app.use(PrimeVue, { ripple: true });
+app.use(VueTippy, {
```

Comment:
Also, since we now have a good implementation of tooltip - do you want to add a tooltip to all the important parts? can we done in a separate PR too. For example, on the editor, the publish button has a tooltip but the home button on its side does not. Ideally, the marker can have a longer tooltip saying that clicking on the marker will select that item and that timestamp can be changed by dragging it, etc.